components/clp-package-utils/clp_package_utils/scripts/decompress.py

```
@@ -354,11 +352,10 @@ def main(argv):
     command = parsed_args.command
     if EXTRACT_FILE_CMD == command:
         return handle_extract_file_cmd(parsed_args, clp_home, default_config_file_path)
-    elif command in (EXTRACT_IR_CMD, EXTRACT_JSON_CMD):
+    if command in (EXTRACT_IR_CMD, EXTRACT_JSON_CMD):
         return handle_extract_stream_cmd(parsed_args, clp_home, default_config_file_path)
-    else:
-        logger.exception(f"Unexpected command: {command}")
-        return -1
+    logger.exception(f"Unexpected command: {command}")
```

Ruff (0.14.5)

Logging statement uses f-string

(G004)